tests/backup/backup_resiliency_test.go

```diff
@@ -132,7 +133,7 @@ var _ = Describe("{BackupRestartPX}", func() {
 				postRuleUid, _ := Inst().Backup.GetRuleUid(orgID, ctx, postRuleNameList[0])
 				backupName := fmt.Sprintf("%s-%s-%v", BackupNamePrefix, namespace, time.Now().Unix())
 				backupNamespaceMap[namespace] = backupName
-				err = CreateBackupWithoutCheck(backupName, SourceClusterName, backupLocation, backupLocationUID, []string{namespace},
+				_, err = CreateBackupWithoutCheck(backupName, SourceClusterName, backupLocation, backupLocationUID, []string{namespace},
```

Comment:
Here can we please add the response also is log messages which will help in debugging in case of failure.
Instead of "_, err = CreateBackupWithoutCheck(backupName, SourceClusterName, backupLocation, backupLocationUID, " you will need to use "response, err = CreateBackupWithoutCheck(backupName, SourceClusterName, backupLocation, backupLocationUID, "

Please change it everywhere it is used

Reply:
I have logged the response regardless of the presence of any errors.

Reply:
The logging of the response has been removed since we are not using it.